Core/Src/IEB_TESTS.c

```diff
@@ -58,13 +57,7 @@ void CHECK_LED_I2C_SPI_TS(void) {
     DBG_PUT("--------------------\r\n");
     DBG_PUT("Testing NAND Flash\r\n");
```

Comment:
might as well yeet these too if we're not actually testing the flash yet

Reply:
I'll just put a debug that says it won't test